src/main/scala-2.11/markets/participants/OrderCanceler.scala

```diff
@@ -15,48 +15,37 @@ limitations under the License.
 */
 package markets.participants
 
-import akka.actor.Scheduler
-
-import scala.concurrent.duration.FiniteDuration
-import scala.concurrent.ExecutionContext
+import markets.{Canceled, Cancel}
+import markets.orders.Order
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1704906892/issues/source?bid=2921361&fileBranchId=3046267#l19)